lib/tapioca/runtime/loader.rb

```diff
@@ -53,9 +53,13 @@ def rails_engines
         return [] unless Object.const_defined?("Rails::Engine")
 
         safe_require("active_support/core_ext/class/subclasses")
+        root = File.expand_path(".")
```

Comment:
As mentioned above, a better heuristic for the root folder is to either use `Rails.root` or `Bundler.default_gemfile`.